### ISSUE TITLE
npm package.json version != github repo version

### ISSUE THREAD

**Issue by rl-hexa** [inxilpro/node-app-root-path#32]
Hi there,

Is there any chance you could bump the package.json version to 2.1.0 so that it matches what is published to npmjs?

I'm trying to get this package approved for use at work. They're pretty strict about licensing and want to link the package publish date back to some commit in the source repo.

The problem here is that version 2.1.0 is what we are grabbing from npm and the publish date of that matches the last commit in the repository, however the version in package.json in this repo indicates 2.0.1.

Basically since there is a mismatch between the source repository and the package contents it is causing issues :(

Many thanks

**Comment by inxilpro** [comment]
Just pushed 2.2.0 to npm and updated the version in Github.
